### ISSUE TITLE
"make install" does not work

### ISSUE THREAD

**Issue by probonopd** [gridcoin-community/Gridcoin-Research#279]
"make install" does not work:

```
$ make INSTALL_ROOT=appdir install ; find appdir/
make: Nothing to be done for `install'.
```

**Comment by probonopd** [comment]
https://travis-ci.org/#L1144-L1147

**Comment by denravonska** [comment]
This is due to missing install files in the project file?

**Comment by probonopd** [comment]
It's probably due to "make install" not being supported by the {q,c}make file.

**Comment by denravonska** [comment]
Fixed with d2c187c0105a96eb3c0f3ad5daf4ce88b1b5b846.
